web/client/components/ScaleBar/ScaleBar.jsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2015, Marko Polovina
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+var React = require('react');
+
+/**
+ * [ScaleBarController new React Controller for ScaleBar]
+ */
+var ScaleBarController = React.createClass({
+    /**
+     * [propTypes types of props]
+     * @type {Object}
+     */
+    propTypes: {
+        scalebox: React.PropTypes.object
+    },
+    /**
+     * [getDefaultProps fetch default props]
+     * @return {[Object]} [return scalebox object]
+     */
+    getDefaultProps() {
+        return {
+            scalebox: {
+                getBarValues: function() {
+                    return ['1:5000', 150];
+                }
+            }
+        };
+    },
+    /**
+     * [getInitialState get initial state]
+     * @return {[Object]} [return empty object]
+     */
+    getInitialState() {
+        return {};
+    },
+    /**
+     * [componentDidMount after component is loaded]
+     * @return {[undefined]} [no return]
+     */
+    componentDidMount() {
+        var that = this;
```

Comment:
I don't personally like the use of that, me and so on. There are better ways to bind functions to this. For example using ES6 arrow functions, or explicity using the bind method.